src/components/App/App.test.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import App from './App.component';
+
+describe('Testing App component', () => {
+  let wrapper;
+  beforeEach(() => {
+    wrapper = render(<App />);
+  });
+
+  it('should to take snapshop', () => {
+    expect(wrapper).toMatchSnapshot();
+  });
+});
```

Comment:
Is on this test the App component fetching the real YouTube Api? seems like that. Remember to avoid real YouTube api calls on tests.